pages/common/mongo.md

```diff
@@ -0,0 +1,19 @@
+# mongo
+
+> MongoDB shell client.
+
+- Connect to a database:
+
+`mongo {{database}}`
+
+- Connect to a database running on a given host on a given port:
+
+`mongo --host {{host}} --port {{port}} {{database}}`
+
+- Connect to a database with a given username, user will be prompted for password:
+
+`mongo --username {{username}} {{database}} --password`
+
+- Run queries on the database:
```

Comment:
`query` is as close to the truth as you're going to get, it's a JS object with limited syntax